src/univariate/continuous/wrappedcauchy.jl

```diff
@@ -0,0 +1,99 @@
+"""
+    WrappedCauchy(r)
+The Wrapped Cauchy distribution with scale factor `r` has probability density function
+```math
+f(x; r) = \\frac{1-r^2}{2\\pi(1+r^2-2r\\cos(x-\\mu))}, \\quad x \\in [-\\pi, \\pi].

```

Comment:
I think I'd rather stick to SciPy, given it's pretty common.

Reply:
As far as I can tell, the scipy implementation does allow for moving the location but through some extra transformation functionality scipy has. Since we don't have that I think it would be nice to keep this within our implementation. I agree the name for the concentration parameter should be changed from 'r' to 'c'.

The VonMises distribution works differently so perhaps we should make it align with that. But that implementation always places it in the centre of the domain which makes things like the cdf rather inflexible (you can't just shift it mod2pi for instance)

Reply:
can also make the support the same as scipy at least i.e. 0<x<2pi

Reply:
> As far as I can tell, the scipy implementation does allow for moving the location but through some extra transformation functionality scipy has. Since we don't have that I think it would be nice to keep this within our implementation. I agree the name for the concentration parameter should be changed from 'r' to 'c'.

You can transform a distribution using `+`, e.g. `2 + Normal()`.